integrations/DCMFloodlight/browser.js

```diff
@@ -285,19 +286,17 @@ class DCMFloodlight {
 
     if (category && name) {
```

Comment:
Ln 287-296 can be simplified.
```
    rudderElement.message.event = `Viewed ${category ? `${category} ` : '' }${name ? `${name} ` : '' }Page`
```

Reply:
@saikumarrs I am pro if else than ternary, it makes the code more understandable. I would go for ternary if the conditions are few

Reply:
I agree, it would also make the statement lengthy. Here, is the updated suggestion.
```
    const catVal = category ? `${category} ` : '';
    const nameVal = name ? `${name} ` : '';
    rudderElement.message.event = `Viewed ${catVal}${nameVal}Page`
```